src/itdelatrisu/opsu/translation/I18n.java

```diff
@@ -72,26 +96,35 @@ private I18n(InputStream stream, String fallbackId) {
 
 		try (BufferedReader reader = new BufferedReader(new InputStreamReader(stream, StandardCharsets.UTF_8))) {
 			String line;
-			while ((line = reader.readLine()) != null) 
-				if (!line.isEmpty() && line.charAt(0) != 35) {
+			while ((line = reader.readLine()) != null) {
+				if (!line.isEmpty() && line.charAt(0) != '#') {
 					String[] values = line.split("=", 2);
 					this.keys.put(values[0], values[1]);
+
+					// Unicode checking
+					if (!unicode)
+						for (char lineChar : values[1].toCharArray())
+							if (lineChar > ASCII_CHAR_LIMIT) {
```

Comment:
There's an alternative way to do this check that is a bit more robust described on [Stack Overflow](https://stackoverflow.com/a/3585247). Make sure to read the comments on the answer as well.